sdk/contentsafety/Azure.AI.ContentSafety/src/Customization/TextBlocklist.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.ContentSafety
+{
+    /// <summary> Text Blocklist. </summary>
+    public partial class TextBlocklist
+    {
+        private string _description;
+        private bool _isDiscriptionChanged = false;
```

Comment:
Here I followed the idea that we use something to track whether a property is set. Since we only have one property need to patch, so I only use a `bool`.